constants/valid-date.ts

```diff
@@ -0,0 +1,31 @@
+export function isValidDateString(value: string): boolean {
```

Comment:
they are all not constant but functions, please put them inside of `lib` and rename this file to `date.ts` to make it more universal